grapple/types/pages.py

```diff
@@ -247,26 +272,36 @@ def on_updated(sender, token, **kwargs):
 
     # Subscription Mixin
     def PagesSubscription():
-        def preview_observable(id, slug, token, content_type):
+        def preview_observable(id, slug, path, token, content_type, site):
```

Comment:
Should we rename `path` to `url_path` for consistency?

Reply:
ah, missed this one. fixed.